<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_forget</class>
 <widget class="QDialog" name="page_forget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#page_forget{background-image: url(:/new/prefix1/1735392184998.jpg);}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>582</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_uesrname">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入（不超过20位）</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>手机号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="le_Id">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>新密码</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="le_newPassword">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入（不超过20位）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>288</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>82</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: #1fab89;
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 10pt;
	font-weight: bold;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>288</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>82</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: #1fab89;
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 10pt;
	font-weight: bold;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
